cmd/dockerd/daemon.go

```diff
@@ -335,8 +358,8 @@ func (cli *DaemonCli) start(opts *daemonOptions) (err error) {
 		return errors.Wrap(err, "shutting down due to ServeAPI error")
 	}
 
-	if sp != nil {
-		sp.Shutdown(context.Background())
+	if tp != nil {
```

Comment:
LOL, I went looking for an `sp` variable in the code (as I was looking per-commit). I guess the original `sp` was a typo for `tp` ?

Reply:
`span processor`
I changed what got returned.

Reply:
Ah! `sp` makes sense now 👍